package.json

```
@@ -1,6 +1,6 @@
 {
-  "name": "doctor-app-manual",
-  "version": "0.1.0",
+  "name": "cra-template-typescript-cardinal-sdk",
```

I think it should be
```suggestion
  "name": "@icure/cra-template-typescript-cardinal-sdk",
```